calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+# mapping_based_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/mapping_based_calibrator.md)
+
+## Purpose
+
+The package `mapping_based_calibrator` allows extrinsic calibration between 3d lidar and 3d lidar sensors, as well as baselink and 3d lidar sensors used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+### lidar-lidar calibration
+
+The calibrator is designed to estimate the transformation between lidar and lidar sensors. It starts by employing a lidar, called the `mapping lidar`, for mapping purposes. Then, it utilizes pointcloud registration algorithms to calibrate the remaining lidars, known as `calibration lidars`, to the `mapping lidar`. Specifically, the calibration process encompasses three primary steps: constructing a map, preparing calibration data, and finally estimating the transformations.
```

Comment:
The lidar is not called that way
We call it that way / treat it as such for the duration of the experiment

Reply:
`calibrate the remaining lidars, known as calibration lidars, to the mapping lidar`

What does calibrating A to B mean

Reply:
@knzo25 
The naming is from your documentation. If you do not like this name, could you suggest what name we should use?

Reply:
You misunderstand the meaning of the previous comment. I will repeat it again:
The lidar is not `called` mapping lidar, we use it as such.